articles/modules/ROOT/pages/setting-up-ssl-with-docker.adoc

```diff
@@ -10,10 +10,141 @@ Neo4j 3.2 added a Unified SSL Framework to setup secure connections for Bolt, HT
 framework can be found at:  https://neo4j.com/docs/operations-manual/current/security/ssl-framework/
 
 Setting up secure Bolt and HTTPS communications when running Neo4j within a Docker Container requires some specific steps and 
-configuration settings:
+configuration settings.
 
 The below steps assume you've followed the process outlined in the above documentation link and have created your public cert and
-private key.
+private key. 
+
+CAUTION: The steps listed below are for example purposes only. You should make sure these are suitable for your requirements before deploying into any environment. 
+
+The configuration steps for Neo4j 3.5 and Neo4j 4.x are different. Both are listed below.
```

Comment:
@umuzammil - In 4.x you need to split out ssl policies by scope (bolt, https, etc..). For example,

`dbms.ssl.policy.<scope>.<setting-suffix>`

In 3.5, you split via ssl policy:

`dbms.ssl.policy.<policy-name>.<setting-suffix>`

With regards to directory structure, In 4.x directories are created in respect to scope. In 3.5, directories are created in respect to ssl policy.

Reply:
@sirReeall essentially the difference is the way we make each communication channel have a distinct namespace in 4.x whereas, in 3.5, the distinction was applied using [<scope>.<policy_name>](https://neo4j.com/docs/operations-manual/3.5/security/ssl-framework/#ssl-policy-apply), in 4.x its applied by <scope> only. This is not to say that there's _no_ difference between versions here, rather what _exactly_ the difference is (perhaps include the detail you mentioned above in this thread?). As you mentioned that following the 3.5 steps in 4.x install wouldn't work. Again, an example of the resulting errors/exceptions would be useful for readers migrating to 4.x. This detail albeit helpful, may however be out of scope of the article, but imo, would help.

Reply:
I'm not sure that level of detail would add any value or fits the purpose of this arctile. If I was looking for this KB article, it would be because I'm trying to setup SSL in either 3.5 or 4.x, rather then trying to understand the difference between the two configurations.

Reply:
nm.. approving for merge